darts/utils/model_selection.py

```diff
@@ -0,0 +1,79 @@
+from typing import Sequence, Optional, Union
+from ..timeseries import TimeSeries
+
+
+def train_test_split(
+        data: Sequence[TimeSeries],
```

Comment:
Could you implement support for `Union[TimeSeries, Sequence[TimeSeries]]` ?

Reply:
done